tools/cli/internal/changelog/outputfilter/message.go

```diff
@@ -0,0 +1,79 @@
+package outputfilter
+
+import (
+	"regexp"
+)
+
+func transformMessage(entry *Entry) *Entry {
+	text := entry.Text
+	text = removeResponseStatusCodes(text)
+	text = setValueSet(text)
+	text = setValueRemoved(text)
+	text = removeInlineSchemaIndex(text)
+	text = removeRedundantOneOfAllOf(text)
+	entry.Text = text
+
+	return entry
+}
+
+func removeRedundantOneOfAllOf(text string) string {
```

Comment:
love how all of this is looking much simpler. to help w/ reviews, can you share which of the transformations from the python scripts  you covered here?

Reply:
this one is from messages.py which I have to say is quite simple also in mms xD

Reply:
ah OK heheheheh lovely and the message.go makes sense too